maze_transformer/generation/constants.py

```diff
@@ -4,6 +4,7 @@
 Coord = NDArray["x y", np.int8]
 CoordTup = tuple[int, int]
 CoordArray = NDArray["coords", np.int8]
+CoordList = list[CoordTup]
```

Comment:
Can we replace these with tuple[int, int] and list[tuple[int, int]]? If not I think there should be two custom types at most rather than 4

Reply:
I'll be refactoring this to unify the types in my next PR, saving that can of worms for later